src/components/TooltipButton/TooltipButton.scss

```diff
@@ -183,27 +190,104 @@ $tooltip-button-size: 46px;
 }
 
 @media #{$smartphone} {
+  $tooltip-button-size--mobile: 52px;
+  $tooltip-buttons-gap--mobile: 16px;
```

Comment:
`$spacing--base` are also 16px. Do you want to reuse that one or have a distinct constant here?